samples/boards/nrf/ieee802154/802154_rpmsg/prj.conf

```diff
@@ -0,0 +1,20 @@
+CONFIG_OPENAMP=y
+CONFIG_OPENAMP_MASTER=n
+CONFIG_OPENAMP_SLAVE=y
+
+CONFIG_IPM=y
+CONFIG_IPM_NRFX=y
+
+CONFIG_IPM_MSG_CH_1_ENABLE=y
+CONFIG_IPM_MSG_CH_1_TX=y
+CONFIG_IPM_MSG_CH_0_ENABLE=y
+CONFIG_IPM_MSG_CH_0_RX=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+
+CONFIG_NRF_802154_SER_RADIO=y
+CONFIG_NRF_802154_SER_DEFAULT_RESPONSE_TIMEOUT=500
+CONFIG_NRF_802154_SL_OPENSOURCE=y
+
+CONFIG_NRFX_RNG=y
```

Comment:
This driver (`nrfx_rng.c`) does not seem to be used by this sample. It looks like this entry is needless, or this option should be enabled by some other component.

Reply:
Good catch! It was not needed at all. Removed.
Also moved `ENTROPY_GENERATOR` to `NRF_802154_RADIO_DRIVER` because it is required by the driver rng platform abstraction implementation. Not the application itself.

Also removed multiple entries from prj.conf that were not used by the sample directly, but were included indirectly by used components.